src/main/java/cl/transbank/patpass/PatpassOptions.java

```diff
@@ -0,0 +1,55 @@
+package cl.transbank.patpass;
+
+import cl.transbank.common.IntegrationType;
+import cl.transbank.common.Options;
+import lombok.*;
+
+@ToString  @AllArgsConstructor
+public class PatpassOptions extends Options implements Cloneable  {
+    @Getter  final String headerCommerceCodeName = "commercecode";
+    @Getter  final String headerApiKeyName = "Authorization";
+
+    public Options buildOptions(Options patpassOptions) {
```

Comment:
Seems like the `buildOptions` logic are same in both `PatpassOptions` and `WebpayOptions` so you could implement it directly in abstract class `Options`

Reply:
done